squad_client/core/models.py

```diff
@@ -183,22 +187,30 @@ def __repr__(self):
 class Build(SquadObject):
 
     endpoint = '/api/builds/'
-    attrs = ['url', 'id', 'testjobs', 'finished',
+    attrs = ['url', 'id', 'testjobs', 'status', 'finished',
              'version', 'created_at', 'datetime', 'patch_id', 'keep_data', 'project',
              'patch_source', 'patch_baseline']
 
+    total_pass_summary = 0
+    total_fail_summary = 0
+    total_skip_summary = 0
+    total_xfail_summary = 0
+
     def testruns(self, count=ALL, bucket_suites=False, **filters):
         filters.update({'build': self.id})
         testruns = self.__fetch__(TestRun, filters, count)
 
         if bucket_suites:
             for _id in testruns.keys():
                 testruns[_id].bucket_metric_and_test_suites()
+                self.total_pass_summary += testruns[_id].total_pass
```

Comment:
This is adding unnecessary API calls. Any reason why we can't use BuildStatus directly?